src/themes/greyVest.js

```diff
@@ -426,14 +426,15 @@ export let GVStyle = () => (
       /* Icon Button */
       .gv-icon-button {
         border-radius: 100px;
-        width: 20px;
-        height: 20px;
```

Comment:
This is only relevant for actual icons but they're already taken into account with `.gv-icon-button i` down below.